sentry/src/main/java/io/sentry/ExperimentalOptions.java

```diff
@@ -0,0 +1,16 @@
+package io.sentry;
+
+import org.jetbrains.annotations.NotNull;
+
+public final class ExperimentalOptions {
```

Comment:
To be honest, I wouldn't add this extra class. Once the feature is stable you'd need to move this over to `Sentry(Android)Options` causing a breaking change. Wouldn't `@Experimental` be enough? At least that's what I did for Metrics 😅

Reply:
yeah, the problem with the `@Experimental` annotation is that it doesn't really highlight it on the consumer's side (i.e. when you use it, the IDE does not highlight that this is experimental), and not everyone uses IDEs that highlight such things. So in order to know about it being experimental we'd have to actually look at the source code.

We wanted to stick to JS pattern which has `experimental` suboptions I think. Alternatively, we could just name the options `ExperimentalSentryReplayOptions` or something like that.

I think it's totally fine to make a breaking change for this, then the users will know that the feature was promoted to stable. We actually want to have `_experimental` only until it's GA'd I believe, and then promote it to SentryOptions, do you think that makes it more plausible?

Reply:
👍 Fine for me, but please add a comment, so it's clear why this is different than usual.